src/SIL.XForge.Scripture/Models/CheckingConfig.cs

```diff
@@ -7,5 +7,6 @@ public class CheckingConfig
         public bool ShareEnabled { get; set; } = false;
         public string ShareLevel { get; set; } = CheckingShareLevel.Specific;
         public string AnswerExportMethod { get; set; } = CheckingAnswerExport.All;
+        public int? NoteTagId { get; set; }
```

Comment:
I am assuming a later change may allow customization of this value?

Reply:
@pmachapman The note tag ID cannot be changed, but its properties can, in Paratext. So the associated icon, name, and more can be changed. We don't plan to add a UI for that. See the spec document for more details around this.